### DISCUSSION TITLE
SUGGEST To Delete Dormant Account

### DISCUSSION THREAD

**Discussion by DennisYun** [community/community#63610] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Thank you so much for providing us programmers wonderful code storage and several services.

A lot of programmers using Github warmly as well as me, however, may complain of an inconvenience, which is that Github users sometimes are not able to use User Name and Org Name that they do want to use. The reason is, clearly, since another user won the account already. If those who were one step faster were using the account ripping, it would be okay. But the problem is 'not everyone'. Some just created the account and has not been using it. Even though some of them may work hard such as creating repos privately, I am sure that most who did not create any repos and whose profile is default thing are not active users. Therefore, I pray earnestly for Github to delete 'these inactive' accounts, just like X(used to be Twitter) did. Thanks.

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @DennisYun ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. I've gone ahead and moved it for you. Good luck!
